package.json

```diff
@@ -89,6 +90,7 @@
     "mocha": "^6.1.4",
     "nodemon": "^2.0.16",
     "npm-run-all": "^4.1.5",
+    "patch-package": "^6.5.0",
```

Comment:
This has to be a top-level dependency because rrweb is hoisted to the root `node_modules` directory, right?

Reply:
yes, this runs as `postinstall` when running `yarn`, so it needs to be on the very top. Also note that the patch itself is also at the root of the project!